config/fcos/v1_6_exp/schema.go

```diff
@@ -32,6 +32,7 @@ type BootDevice struct {
 
 type BootDeviceLuks struct {
 	Discard   *bool       `yaml:"discard"`
+	Device    *string     `yaml:"s390x-device"`
```

Comment:
```suggestion
	Device    *string     `yaml:"blockdev"`
```

Reply:
Is prefix `s390x-` help in identifying what layout the user trying to use and avoid confusion on device parameter from other arch?